app/src/main/java/dk/nodes/arch/extensions/CoroutinesEx.kt

```diff
@@ -0,0 +1,36 @@
+package dk.nodes.arch.extensions
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.async
+import kotlinx.coroutines.launch
+import java.util.concurrent.TimeUnit
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+
+suspend fun delay(time: Long, timeUnit: TimeUnit) {
+    kotlinx.coroutines.delay(timeUnit.toMillis(time))
+}
+
+fun launch(
+        context: CoroutineContext = EmptyCoroutineContext,
+        block: suspend CoroutineScope.() -> Unit
+) = GlobalScope.launch(context = context, block = block)
+
+fun launchOnUI(block: suspend CoroutineScope.() -> Unit) =
+    launch(context = Dispatchers.Main, block = block)
+
+fun launchOnIO(block: suspend CoroutineScope.() -> Unit) =
+    launch(context = Dispatchers.IO, block = block)
+
+fun <T> async(
```

Comment:
This is fine for the mean time for MVP projects that will upgrade to kotlin 1.3, but please note that this shouldn't be used in the new template project where coroutine scope is passed down from the viewmodels (and thereby cancelled when viewmodel detaches)